.go-version

```diff
@@ -1 +1 @@
-1.22.8
+1.23.3
```

Comment:
I think 1.23.3 has CVEs associated with it; I'd suggest going with the latest 1.23.